vllm/entrypoints/openai/api_server.py

```diff
@@ -449,6 +450,13 @@ async def authentication(request: Request, call_next):
                                     status_code=401)
             return await call_next(request)
 
+    @app.middleware("http")
+    async def add_request_id(request: Request, call_next):
+        request_id = request.headers.get("X-Request-Id", uuid.uuid4().hex)
+        response = await call_next(request)
+        response.headers["X-Request-Id"] = request_id
+        return response
```

Comment:
This `X-Request-id` seems not connected to `request_id` in (`create_chat_completion`)[https://github.com/vllm-project/vllm/blob/main/vllm/entrypoints/openai/serving_chat.py#L179].

Reply:
Right, this id has nothing to do with chat request id(or request id of other OpenAI API response body). This behavior is consistent with OprnAI's current behavior, and you can recover this id like:

```python
from openai import OpenAI

client = OpenAI(...)
response = client.chat.completions.create(...)
response._request_id
```

Reply:
Then, LGTM.